contractsV5/KyberNetwork.sol

```diff
@@ -675,10 +675,85 @@ contract KyberNetwork is Withdrawable, Utils, IKyberNetwork, ReentrancyGuard {
         }
     }
     
+    event HandlePlatformFee(address indexed recipient, uint fees);
+
     function handleFees(TradeData memory tradeData) internal returns(bool) {
         // create array of reserves receiving fees + fee percent per reserve
         // fees should add up to 100%.
         // send total fee amount to fee handler with reserve data.
+
+        address[] memory eligibleWallets = new address[](tradeData.numFeePayingReserves);
+        uint[] memory rebatePercentages = new uint[](tradeData.numFeePayingReserves);
+
+        // Updates reserve eligibility and rebate percentages
+        updateEligibilityAndRebates(eligibleWallets, rebatePercentages, tradeData);
+
+        // Sending platform fee to taker platform
+        (bool success, ) = tradeData.platformFeeWei != 0 ?
+            tradeData.input.platformWallet.call.value(tradeData.platformFeeWei)("") :
+            (true, bytes(""));
+        require(success, "Transfer platform fee to taker platform failed");
+        emit HandlePlatformFee(tradeData.input.platformWallet, tradeData.platformFeeWei);
+
+        // Send total fee amount to fee handler with reserve data.
+        require(
+            feeHandlerContract.handleFees.value(tradeData.networkFeeWei)(eligibleWallets, rebatePercentages),
+            "Transfer network fee to FeeHandler failed"
+        );
+        emit HandlePlatformFee(address(feeHandlerContract), tradeData.networkFeeWei);
```

Comment:
actually I was thinking you are reporting that u send fee to platform address.

we don't need to issue an event for this function call.
who will use it?

The event is important for the fact we sent eth to platform wallet. and they will use it.
so address should be platform wallet.

later in fee hander can issue event that we updated rebate wallets internal account with fees.

Reply:
please remove this event.

Reply:
it is already removed.

Reply:
I see this line in the PR
line 703
emit HandlePlatformFee(address(feeHandlerContract), tradeData.networkFeeWei);


so where is it removed?